manta-accounting/src/transfer/mod.rs

```diff
@@ -100,10 +100,10 @@ pub trait Configuration {
     type Compiler: Assert;
 
     /// Asset Id Type
-    type AssetId: Clone + Ord;
+    type AssetId: Clone + Ord + Debug;
 
     /// Asset Value Type
-    type AssetValue: AddAssign + Clone + Default + PartialOrd + Sum;
+    type AssetValue: AddAssign + Clone + Default + PartialOrd + Sum + Debug;
```

Comment:
No need for `Debug` here either.

Reply:
Doesn't the javascriptcompatability macro require it or atheist FMT

Reply:
I remember previous it compiled error on signer, so add `Debug` trait. But now I remove `Debug` trait, and it compiled ok now.